cli/setup.py

```diff
@@ -1,8 +1,9 @@
 from setuptools import setup
+import popper
 
 setup(
     name='popper',
-    version='0.6-dev0',
+    version='popper.__version__',
```

Comment:
This is not going to work with setuptools. One way you can try is by doing the following:

```
cd cli/
python setup.py sdist
```

you should see error and warning messages. The referenced links in issue #221 give descriptions of what can be done for single-sourcing the version number, for example, alternative number 3 of the [official packaging documentation on this](https://packaging.python.org/guides/single-sourcing-package-version/) describes a way of using `execfile` to load the `__init.py__` file from `setup.py`.